src/main/java/com/minecolonies/coremod/inventory/InventoryCitizen.java

```diff
@@ -913,7 +913,7 @@ public NBTTagList writeToNBT(NBTTagList nbtTagListIn)
         {
             if (!(this.offHandInventory.get(k)).isEmpty())
             {
-                NBTTagCompound nbttagcompound2 = new NBTTagCompound();
+                final NBTTagCompound nbttagcompound2 = new NBTTagCompound();
                 nbttagcompound2.setByte("Slot", (byte) (k + 150));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 150 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)